### DISCUSSION TITLE
how to add a DNS server

### DISCUSSION THREAD

**Discussion by Pina1924** [DNSCrypt/dnscrypt-proxy#2450] [Q&A]
hi, i'd like to use also:
https://dns.watch/
but i see it is not in the list:
https://dnscrypt.info/public-servers/
so how to modify my .toml file in order to use also this DNS server...?
at this time my .toml file has this line:

`server_names = ['quad9-doh-ip4-port443-filter-pri', 'cloudflare', 'cloudflare-security', 'meganerd-doh-ipv4', 'uncensoreddns-ipv4']`

thxs so much in adv

cheers

**Comment by ghost** [comment]
DNSCrypt-Proxy only supports encrypted DNS (DNSCrypt and DoH) but dns.watch doesn't currently provide that, so you cannot use it.

**Comment by Pina1924** [comment]
Hi, first thxs
how about yr very strange nickname...
https://github.com/c5aba924-b91a-4504-b8e7-9dd81a704703 ??...:(
so you state that when dns.watch provide dnscrypt and doh it will be added to the :
https://dnscrypt.info/public-servers/
and i will be able to modify the .toml config file in order to be able to use also it...??
thxs in adv
cheers

**Comment by ghost** [comment]
> so you state that when dns.watch provide dnscrypt and doh it will be added to the :
https://dnscrypt.info/public-servers/
and i will be able to modify the .toml config file in order to be able to use also it...??

Yes, but who knows if and when dns.watch does that.
